examples/tomcat-operator/rbac.yaml

```diff
@@ -19,6 +19,7 @@ rules:
   - events
   - configmaps
   - secrets
+  - namespaces
```

Comment:
Why does a Role need permissions for `namespaces`? That's a cluster-scoped resource. We'd need a ClusterRole for that.
Also I think the helm-operator won't need a ClusterRole to watch all namespaces since we restrict the manager to a single namespace.
https://github.com/operator-framework/helm-app-operator-kit/blob/master/helm-app-operator/cmd/manager/main.go#L58

Reply:
@hasbro17 It's a quirk (feature?) of the tiller kube client, which ensures that the namespace exists during release installation. See https://github.com/helm/helm/blob/master/pkg/kube/client.go#L92.

Adding namespace permissions in a Role looks like it gives users with that role only permissions on the namespace in which the Role itself exists.

I made a "foo" namespace with this Role and RoleBinding to test what works and doesn't work with namespace permissions:
```
$ kubectl get namespaces
Error from server (Forbidden): namespaces is forbidden: User "system:serviceaccount:foo:default" cannot list namespaces at the cluster scope

$ kubectl get namespace foo
NAME   STATUS   AGE
foo    Active   5m

$ kubectl get namespace default
Error from server (Forbidden): namespaces "default" is forbidden: User "system:serviceaccount:foo:default" cannot get namespaces in the namespace "default"

$ kubectl create namespace bar
Error from server (Forbidden): namespaces is forbidden: User "system:serviceaccount:foo:default" cannot create namespaces at the cluster scope
```

Here's one that DOES work that probably shouldn't:
```
$ kubectl delete namespace foo
namespace "foo" deleted
```

Based on what the client is trying to do, it looks like namespaces permissions in the Role can be restricted to only the get verb. WDYT?